src/main/java/com/chargerrobotics/subsystems/LimelightSubsystem.java

```diff
@@ -94,9 +241,14 @@ public void periodic() {
 		x = tx.getDouble(0.0);
 		y = ty.getDouble(0.0);
 		v = tv.getDouble(0.0);
+		s = ts.getDouble(0.0);
 		SmartDashboard.putNumber("LimelightX", x);
 		SmartDashboard.putNumber("LimelightY", y);
+		SmartDashboard.putNumber("LimelightS", s);
 		SmartDashboard.putNumber("Limelightdistance", distance());
```

Comment:
Since `distance()` can now return `null` (since it's returning a `Double`), this call can throw a `NullPointerException`. You should fetch the value, check to see if it is null, and in this case, put some user readable obvious value representing that it is invalid.

I would also check other locations that you call `distance()` or similar methods that can return a nullable value.

As a note, I noticed this since the `FT_ShooterOnCommand` test failed [as seen here](https://github.com/KentridgeRobotics/2021Robot/pull/23/checks?check_run_id=1944558792#step:4:43)